sdks/go/container/boot.go

```diff
@@ -104,7 +104,7 @@ func main() {
 
 	switch len(artifacts) {
 	case 0:
-		log.Fatal("No artifacts staged")
+		log.Printf("No artifacts staged. Attempting to run default \"worker\".")
```

Comment:
Shouldn't this be fatal? There isn't a "default" worker.

Reply:
I can revert this line. It was mostly to explore if the artifacts got copied to the expected location *anyway* because the model was discovered that way in the first place.